distributed/worker.py

```diff
@@ -1891,7 +1891,7 @@ def update_data(
 
                 try:
                     recs = self._put_key_in_memory(ts, value, stimulus_id=stimulus_id)
-                except Exception as e:
+                except Exception as e:  # pragma: no cover
```

Comment:
This should be tested

```suggestion
                except Exception as e:
```